### DISCUSSION TITLE
can't set "After how many seconds after page load show status widget"

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1875] [General]
it seems that this setting does not work. you can find it in `Edit - Widget themes` under the tab `status widget style`.
![image](https://user-images.githubusercontent.com/13809377/210826139-07b17ba2-1a02-4194-8b55-3f3bd180b88a.png)

**Comment by remdex** [comment]
This is not support for a new widget at the moment. If you can it you can sponsor of adding support it for a new widget.

**Comment by remdex** [comment]
This was implemented. Please update from master branch. There is also now possible to pass this directly as status_delay argument.
